website/docs/tutorials/basic/running_your_app/3_working_directory.md

```diff
@@ -54,22 +56,42 @@ Inside the Hydra output directory we have:
 And in the main output directory:
 * `my_app.log`: A log file created for this run
 
-### Disable changing current working dir to job's output dir
+You can configure the name of the output directory using
+the [customizing the working directory](/configure_hydra/workdir.md) pattern.
+
+
+### Enable/disable changing current working dir to job's output dir
+
+By setting `hydra.job.chdir=True`, you can configure
+Hydra's `@hydra.main` decorator to change python's working directory by calling
+`os.chdir` before passing control to the user's decorated main function.
+As of Hydra v1.2, `hydra.job.chdir` defaults to `False`.
+Setting `hydra.job.chdir=True` enables convenient use of the output directory to
+store output for the application (For example, a database dump file).
 
-By default, Hydra's `@hydra.main` decorator makes a call to `os.chdir` before passing control to the user's decorated main function.
-Set `hydra.job.chdir=False` to disable this behavior.
 ```bash
 # check current working dir
-$ pwd  
-/home/omry/dev/hydra
+$ pwd
+/home/jasha/dev/hydra
+
+# working dir changed to output dir
+$ python my_app.py hydra.job.chdir=True
+Working directory : /home/jasha/dev/hydra/outputs/2023-04-18/13-43-17
+Output directory  : /home/jasha/dev/hydra/outputs/2023-04-18/13-43-17
 
-# working dir remains the same
+# working dir remains unchanged
 $ python my_app.py hydra.job.chdir=False
-Working directory : /home/omry/dev/hydra
+Working directory : /home/jasha/dev/hydra
+Output directory  : /home/jasha/dev/hydra/outputs/2023-04-18/13-43-20
+
```

Comment:
I would eliminate this block.
Show the default case and the case where you override chdir to True.